dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -591,7 +590,9 @@ private static Set<Author> getAndCheckAuthorsForMetadataCreator(Workflow workflo
         }
 
         if (setOfAuthors.isEmpty()) {
-            throw new CustomWebApplicationException(AT_LEAST_ONE_AUTHOR_IS_REQUIRED_TO_PUBLISH_TO_ZENODO, HttpStatus.SC_BAD_REQUEST);
+            Author fillerAuthor = new Author();
```

Comment:
Quick discussion: Can we do better by using the github organization name rather than "Author not specified"? On the other hand, do we want to or just encourage the use of a real author?

Reply:
Changed the code to use the raw organization name as the filler author name.